mozetl/search/aggregates.py

```diff
@@ -126,8 +127,10 @@ def agg_search_data(main_summary, grouping_cols, agg_functions):
     # Do all aggregations
     aggregated = (
         augmented
+        .withColumn('hll', expr('hll_create(client_id, 12)'))
```

Comment:
Can we do this to the `main_summary` object at line 91 to avoid modifying the general `agg_search_data` function?